command/read.go

```diff
@@ -57,6 +59,9 @@ func (c *ReadCommand) AutocompleteFlags() complete.Flags {
 }
 
 func (c *ReadCommand) Run(args []string) int {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
Not sure your thoughts, but would we want to respect the value from `VAULT_CLIENT_TIMEOUT`?

Reply:
Good catch 👍 , I've added a fix